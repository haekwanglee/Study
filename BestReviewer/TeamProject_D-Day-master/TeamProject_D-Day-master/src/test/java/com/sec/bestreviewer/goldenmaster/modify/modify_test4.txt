SCH,-p, , ,employeeNum,90000000
MOD,-p, , ,employeeNum,90000000,name,Modified Name
SCH,-p,-m, ,birthday,01
MOD,-p,-m, ,birthday,01,certi,EX
SCH,-p,-m, ,birthday,01
SCH,-p,-y, ,birthday,1997
MOD,-p,-y, ,birthday,1997,certi,EX
SCH,-p,-l, ,name,AAA
MOD,-p,-l, ,name,AAA,phoneNum,010-0000-0000