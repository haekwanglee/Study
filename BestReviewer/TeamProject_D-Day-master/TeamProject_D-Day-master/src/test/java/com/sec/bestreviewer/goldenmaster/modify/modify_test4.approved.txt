SCH,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
MOD,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
SCH,90000000,Modified Name,CL2,010-1111-1111,19980101,ADV
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
MOD,90000000,Modified Name,CL2,010-1111-1111,19980101,ADV
MOD,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
MOD,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
MOD,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
MOD,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
SCH,90000000,Modified Name,CL2,010-1111-1111,19980101,EX
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,EX
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,EX
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,EX
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,EX
SCH,NONE
MOD,NONE
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,EX
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,EX
SCH,00210001,DICTIONARY AAA,CL2,010-3333-1113,19980303,ADV
MOD,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,EX
MOD,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,EX
MOD,00210001,DICTIONARY AAA,CL2,010-3333-1113,19980303,ADV
